omod/src/test/java/org/openmrs/module/fhir2/narrative/DefaultNarrativeGeneratorForTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.narrative;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.narrative.CustomThymeleafNarrativeGenerator;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.hl7.fhir.r4.model.Narrative;
+import org.openmrs.module.fhir2.FhirConstants;
+
+public class DefaultNarrativeGeneratorForTest {
```

Comment:
Why is this a separate class?

Reply:
Actually I thought to keep the generator separate. But I'll move it to the same class with the tests.